test/integration_test/migration_test.go

```diff
@@ -30,17 +31,23 @@ const (
 	operatorRestartRetryInterval = 10 * time.Second
 	clusterCreationTimeout       = 2 * time.Minute
 	clusterCreationRetryInterval = 5 * time.Second
+	labelKeyPXEnabled            = "px/enabled"
 )
 
 var migrationTestCases = []types.TestCase{
 	{
-		TestName:   "SimpleDaemonSetMigration",
+		TestName:   "SimpleMigration",
 		ShouldSkip: func() bool { return false },
```

Comment:
nit: we could remove this as it will be set to return false by default if missing.